### PR TITLE
More docs for running dask on NCAR systems

### LINKED ISSUE #37: Document using dask-distributed on Geyser/Caldera Systems
With Cheyenne's prolonged outage, I have been working on Geyser using a local cluster with a fair bit of success. I'll just share the workflow here so others can move forward in the short term. 

1. log on to yellowstone
   ```$ ssh yellowstone.ucar.edu```
2. go to your notebook directory, for me that was:
    ```$ cd ~/projects/pangeo/notebooks```
3. launch a notebook server on a geyser node:
    ```execgy notebook```
4. follow the instructions for creating an ssh tunnel, something like:
    ```ssh -N -l user_name -L 8888:geyserXX-ib:8888 yellowstone.ucar.edu```
5. Open a notebook and connect to a dask distributed "local cluster" using:
    ```Python
    from distributed import LocalCluster, Client
    client = Client(LocalCluster(processes=True, n_workers=10, threads_per_worker=2))
    client
    ```

cc @rabernat - this could be useful to you.
cc @mrocklin  - any thoughts on how this should perform. Seems to be working well for me.

### PR DESCRIPTION
**Pull request by jhamman**
standby for short a short blip on running on geyser/caldera.

closes #37

### CODE REVIEW

File: docs/setup_guides/cheyenne.rst [hunk 1]
```diff
@@ -114,6 +114,39 @@ Great. We're done with that.
 Launch Dask with a script
 -------------------------
 
+.. note::
+
+   The following scripts and proceedures have been packed into a convenient wrapper
+   script ``launch-dask.sh``. It and its supporting utilities can be found in the
+   `pangeo Github reposository <https://github.com/pangeo-data/pangeo/tree/master/utilities/cheyenne>`__.
+
+   The ussage of this script is quite simple:
```
Comment (rabernat) [inline docs/setup_guides/cheyenne.rst]: spelling

**Comment by jhamman** [comment]
cc @msaharia.

**Comment by rabernat** [comment]
This is great Joe! Looks good to merge. Or do you have more updates coming?

**Comment by mrocklin** [comment]
+1 from me

**Comment by jhamman** [comment]
Thats all for now.

**Pull request merged**